examples/remix-cms/app/utils/markdown.server.ts

```diff
@@ -1,4 +1,5 @@
 import { marked } from "marked";
+// TODO: Find a lighter solution for syntax highlighting
```

Comment:
Not related to ALS per se, but my CF Workers bundle is like 4MB because of this 🙃